index.js

```diff
@@ -37,9 +37,9 @@ var multipartser = function multipartser () {
     if ( splits.length < 3 ) { return; }
 
     // we found at least one part
-    // the first element should be empty string indicating that 
+    // the first element should be empty string indicating that
     // boundary was at the beginning of the body
-    if ( splits[ 0 ] != '' ) {
+    if ( !splits[ 0 ].match(/^\s+$/) ) {
```

Comment:
Thanks for responding quickly :)

I'd be happy with that, though the other side of the argument is the robust software principal "Be conservative in what you do, be liberal in what you accept from others" *shrug*